lesson2/README.md

```diff
@@ -1,14 +1,85 @@
 # Lesson 2 - Android Basics
-
+### Your first Android application!
 Today in class we'll walk you through making your first Android application. Found in the lesson2 folder is a copy of the application that we will create in class.
 
-## [Activities](https://developer.android.com/guide/components/activities.html) and [Fragments](https://developer.android.com/guide/components/fragments.html)
+## Create a Blank App
+To begin with, we'll create a default app and walk you through the different components.
 
-more info
+1. Create a new project in Android Studio
+2. Name it something akin to "Lesson2HW" and save it in your forked repo
+3. Click "Basic Activity"
+4. Check "Use a Fragment"
 
+Great, you just created your very first app!
 
```

Comment:
I was gonna describe the layout of files first, then explain how to run apps down below (it's not intuitive how to enable debugging mode and connect your phone, etc.)